components/logging/org.wso2.carbon.logging.view.ui/src/main/resources/org/wso2/carbon/logging/view/ui/i18n/Resources.properties

```diff
@@ -42,4 +42,4 @@ view.logs.within.the.past.thirty.mins=Logs taken within past thirty minutes.
 view.logs.between=Logs taken between.
 log.level=Level
 current.time=Current Time Zone:
-

```

Comment:
What if you say, "Message, truncated due its large size.", wouldn't that be much more informative and understandable rather than saying "more" ?